### PR TITLE
Kb ui reorg

### LINKED ISSUE #49: No 'customer has no payment options' msg printing
### Context
1. Selecting a customer, who has an open order, and selecting `6` should continue on with the checking out process
2. If that customer has an open _but no payment options_, then 'Customer has no payment options' should be printed.

### Associated Product Ticket
#6 

### Process
1. Check out the logic in `controllers/completeOrderC`, especially around lines `31-34`
1. Check out `ui.js` to make sure logs the resolved string

### Expected Result
1. `Customer has no payment options` should print before the main menu reprints

### PR DESCRIPTION
**Pull request by kenziebottoms**
# Description
`ui.js` has been refactored to work more like a game loop. Each controller function `return`s/`reject`s a response message which is dealt with and printed nicely in `ui.js`. All controllers can be dealt with this way except for `addCustomerPaymentOption`, which is a two-step process. It may not have to be; refactor later.

## Related Ticket(s)
All of em. Fixes #49 

## Problem to Solve
`ui.js` deals with all the error/success messages in one place in one way.

## Expected Behavior
- You should never get kicked back to the main menu without a red error message.
- All successful interactions should be acknowledged with a green success message.

## Steps to Test Solution
Test all of it. Go find closed PRs if you don't remember how one aspect was tested.

## Testing
- [x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
- [x] I certify that all existing tests pass

## Documentation
- [ ] There is new documentation in this pull request that must be reviewed..
- [ ] I added documentation for any new classes/methods

### CODE REVIEW

File: app/views/completeOrderV.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports.createPrompt = (total, opts) => {
       pattern: optionsRegEx,
```
Comment (joechesney) [inline app/views/completeOrderV.js:22]: description: `\nAvailable Payment Options:\n${optionsMsg}\nChoose a payment option`,

Comment (joechesney) [reply app/views/completeOrderV.js:22]: i think the description on line 21 here, should be the above statement

**Review by joechesney — DISMISSED** [review]
hella chuuups

**Review by jordan-castelloe — APPROVED** [review]
THIS IS A THING OF BEAUTY

**Review by melissabell456 — APPROVED** [review]
40 tests passing, all prompts working with proper functionality

**Review by joechesney — APPROVED** [review]
mega ultra chuuuuppppss

**Pull request merged**